cli/client.go

```diff
@@ -1294,6 +1298,47 @@ type machinesPartCopyFilesArgs struct {
 	NoProgress   bool
 }
 
+type wrongNumArgsError struct {
+	have int
+	min  int
+	max  int
+}
+
+func (err wrongNumArgsError) Error() string {
+	if err.min != err.max && err.max == 0 {
+		noun := "arguments"
+		if err.min == 1 {
+			noun = "argument"
+		}
+		return fmt.Sprintf("expected %d %s but got %d", err.min, noun, err.have)
+	}
+	return fmt.Sprintf("expected %d-%d arguments but got %d", err.min, err.max, err.have)
+}
+
+type machinesPartGetFTDCArgs struct {
+	Organization string
+	Location     string
+	Machine      string
+	Part         string
+	NoProgress   bool
```

Comment:
(q) why is this part of the args? It's not an actual argument to the command and doesn't seem to be referenced anywhere.

Reply:
Oops, good catch. It initially did have a flag for this but during testing I noticed it didn't change anything about the behavior so removed it from the flags but forgot about this field in the struct. I'm actually a bit curious about whether the same flag on `machine part cp` is vestigial since the help text for that command says "Note: There is no progress meter while copying is in progress."

Reply:
Hmm interesting! I think @michaellee1019 would be able to speak to this with better confidence but it looks to me like that note is what should be removed, not the flag, as it looks to me that we do respect the `no-progress` flag in `copyFilesToMachine`